### PR TITLE
Implement HuggingFaceVisualQuestionAnsweringRemoteInferencePromptSchema

### PR DESCRIPTION
**Pull request by rholinshead**
Implement HuggingFaceVisualQuestionAnsweringRemoteInferencePromptSchema

# Implement HuggingFaceVisualQuestionAnsweringRemoteInferencePromptSchema

For https://github.com/lastmile-ai/aiconfig/issues/1392

This will add the prompt schema so that visual question answering prompts have the nice UI for input and settings

---
[//]: # (BEGIN SAPLING FOOTER)
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/1396).
* #1397
* __->__ #1396

### CODE REVIEW

**Review by Ankush-lastmile — APPROVED** [review]
lgtm, we might want to revisit how text + other modality input box renders

Screenshot 2024-03-04 at 4 56 29 PM

File: python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceVisualQuestionAnsweringRemoteInferencePromptSchema.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      },
+    },
+    model_settings: {
+      type: "object",
+      properties: {
+        model: {
+          type: "string",
+          description: `Hugging Face model to use. Can be a model ID hosted on the Hugging Face Hub or a URL 
+        to a deployed Inference Endpoint`,
+          default: "dandelin/vilt-b32-finetuned-vqa",
```
Comment (Ankush-lastmile) [inline python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceVisualQuestionAnsweringRemoteInferencePromptSchema.ts:38]: does this model work fine on the free inference endpoint?

Comment (rholinshead) [reply python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceVisualQuestionAnsweringRemoteInferencePromptSchema.ts:38]: Yep - 
Screenshot 2024-03-04 at 5 01 40 PM

**Comment by rholinshead** [comment]
> lgtm, we might want to revisit how text + other modality input box renders
> 
> Screenshot 2024-03-04 at 4 56 29 PM

Ya, I think we can improve it a bit. I wonder if we should just make it ordered by the properties as well? Make sense to have image first in this case

**Pull request merged**